src/cp.js

```diff
@@ -260,9 +261,25 @@ function _cp(options, sources, dest) {
         return; // skip file
       }
 
+      if (path.relative(src, thisDest) === '') {
+        // a file cannot be copied to itself, but we want to continue copying other files,
+        // so save the error for later
+        identicalFiles.push("'" + thisDest + "' and '" + src + "' are the same file");
```

Comment:
It sets `result.stderr` to the error output (it's a `+=` operation, so it all collects up). `shell.error()` is some truthy value (it's currently the same string as `result.stderr`, but this is no longer guaranteed as part of the API and it may later change to an error code).